### ISSUE TITLE
Guidelines issue

### ISSUE THREAD

**Issue by dfvgbh** [salesforce-ux/design-system#628]
> (REQUIRED) Include one or more screenshots if applicable, as well as a Codepen with the
reduced test case.

https://www.lightningdesignsystem.com/accessibility/guidelines/global-focus/#toasts
![image](https://user-images.githubusercontent.com/17325859/67947405-44678780-fbec-11e9-8d63-3365c7fcb6ae.png)

https://www.lightningdesignsystem.com/components/toast/#Accessibility
![image](https://user-images.githubusercontent.com/17325859/67947428-5517fd80-fbec-11e9-93e2-3215236fa34a.png)

> Describe the issue. Is it a bug or a feature request (new component, new icon, new CSS class)?

Missaligned role attributes for Toasts on different pages.

> Is this issue related to a specific component, variant, and/or state? If so, please detail which.

A11y Focus guidelines => Toast section

> Are any specific browsers impacted by this bug?

No

> Which version of the Salesforce Lightning Design System are you using?

Site: Version 1.2.8 | Framework: Version 2.10.0

> What steps and/or code are needed to reproduce this issue?

> What did you expect to happen?

"role=status" here https://www.lightningdesignsystem.com/accessibility/guidelines/global-focus/#toasts

> What actually happened?

"role=alert" here https://www.lightningdesignsystem.com/accessibility/guidelines/global-focus/#toasts

**Comment by alexiscordova** [comment]
@dfvgbh We'll take a look at this. Thanks for the heads up

**Comment by alexiscordova** [comment]
This should be resolved now:
https://www.lightningdesignsystem.com/accessibility/guidelines/global-focus/#toasts
https://www.lightningdesignsystem.com/components/toast/
